nawab.py

```diff
@@ -43,6 +44,25 @@ def wrapper(func, args, res):
 
 
 def main():
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument("-r", "--retweet", help="Retweet all tweets automatically, doesn't spawn a telegram bot",
+                        action='store_true', required=False)
+    parser.add_argument("-t", "--tg_bot", help="Doesn't retweet all automatically, can manually retweet through telegram bot",
+                        action='store_true', required=False)
+    parser.add_argument("-b", "--blacklist",type=str, help="Blacklist the given username",)
+    parser.add_argument('-l', '--level', action='count', default=1)
+    parser.add_argument('-h', '--help', action='help', default=argparse.SUPPRESS,
+                        help='Show this help message and exit.')
+    parser.add_argument('-v', '--version', action='version',
+                        version='%(prog)s 1.0', help="Show program's version number and exit.")
+
+    args = vars(parser.parse_args())
```

Comment:
If you wish to call arguments for example, blacklist argument like this: `args.blacklist` , then remove this line.